pkg/pgmodel/end_to_end_tests/migrate_test.go

```diff
@@ -40,3 +44,151 @@ func TestMigrateTwice(t *testing.T) {
 		performMigrate(t, connectURL)
 	})
 }
+
+func verifyLogs(t testing.TB, db *pgxpool.Pool, expected []string) {
+	rows, err := db.Query(context.Background(), "SELECT msg FROM log ORDER BY id")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	found := make([]string, 0)
+	for rows.Next() {
+		var value string
+		err = rows.Scan(&value)
+		if err != nil {
+			t.Fatal(err)
+		}
+		found = append(found, value)
+	}
+	if !reflect.DeepEqual(expected, found) {
+		t.Errorf("wrong values in DB\nexpected:\n\t%v\ngot:\n\t%v", expected, found)
```

Comment:
I've found in the pgx tests that for anything with control characters (such as newlines) it pays to go through the array line-by-line and print each as a string; that way on an error you can put both versions into a diff too and it'll tell you exactly where the difference is. I've found that wen everything is on a single line, like here, the output is much less readable.